### PR TITLE
Update code review and rules documents

### PR DESCRIPTION
**Pull request by oliverjam**
- [x] Re-order Gitbook nav
- [x] Code review guide
    - Mostly removed italics/emojis and made it more concise
- [x] Rules
    - Rename from house-rules
    - Clarify everything

I will open a separate PR with potential additions to the rules (based on Recurse Center's social rules)

### CODE REVIEW

File: documents/code-review.md [hunk 1]
```diff
@@ -1,85 +1,66 @@
...
 
-### 3. Raise issues :space_invader:
+Open the project in your editor. Does the file structure make sense? Can you you follow the different paths the code might take? Do variables and functions have clear and descriptive names?
 
-- Use the search bar to see whether the issue you want to raise already exists
-  ![Looking at issues](https://files.gitter.im/Jen-Harris/zjN6/image.png)
+### Raise issues
 
-- Raise issues on their repo if you spot any gaps in clarity in the code or documentation.
+Check if the issue you want to raise already exists. Use the search bar if there are lots of issues already. Make sure your issue titles clearly summarise the content. If you're comment on a specific section of code you can [embed it in the issue](https://help.github.com/en/github/managing-your-work-on-github/opening-an-issue-from-code). Link to relevant documentation if you're suggesting something they may not have seen before.
```
Comment (Albadylic) [inline documents/code-review.md]: If you're comment -> If you comment / If you're commenting

File: documents/rules.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# Rules
+
+## Course rules
+
+### Arrive on time
+
+Arrive **before scheduled start time**. If you're still in the kitchen making coffee when a workshop is supposed to begin you aren't on time, you're late. The mentors will not wait for everyone to be present before starting in the morning.
+
+Let your cohort or the course facilitator know in advance if you're going to be away. Lots of people are giving freely of their time and knowledge to make FAC possible and giving up other commitments to be here.
```
Comment (Albadylic) [inline documents/rules.md]: Lots of people are freely giving their time

Comment (oliverjam) [reply documents/rules.md]: Ha the original wording of this doc was quite unusual

File: documents/code-review.md (hunk 1 above)
Comment (oliverjam) [reply documents/code-review.md]:
```suggestion
Check if the issue you want to raise already exists. Use the search bar if there are lots of issues already. Make sure your issue titles clearly summarise the content. If you're commenting on a specific section of code you can [embed it in the issue](https://help.github.com/en/github/managing-your-work-on-github/opening-an-issue-from-code). Link to relevant documentation if you're suggesting something they may not have seen before.
```

File: documents/rules.md (hunk 2 above)
Comment (oliverjam) [reply documents/rules.md]:
```suggestion
Let your cohort or the course facilitator know in advance if you're going to be away. Lots of people are freely giving their time and knowledge to make FAC possible and giving up other commitments to be here.
```

**Pull request merged**